README.adoc

```diff
@@ -169,20 +165,18 @@ receives the message that contains the hostname and the average system load. The
 the method retrieves the message by connecting to the `systemLoad` channel in
 Kafka, which you configure in the next section.
 
-Each time a message is received, the [hotspot=getSystemLoadMessage
-file=0]`getSystemLoadMessage()` method is called, and the hostname and system
+Each time a message is received, the [hotspot=getSystemLoadMessage file=0]`getSystemLoadMessage()` 
+method is called, and the hostname and system
 load contained in that message are broadcasted in an event to all subscribers.
 
 === Broadcasting events
 
-Broadcasting events is handled in the [hotspot=broadcastData
-file=0]`broadcastData()` method. First, it checks whether the
-[hotspot=broadcaster file=0]`broadcaster` value is [hotspot=notNull file=0]`null`.
-There must be at least one subscriber or there's no
-client to send the event to. If the `broadcaster` value is specified, the
-`OutboundSseEvent` interface is created by using the [hotspot=newEventBuilder
-file=0]`Sse.newEventBuilder()` method, where the [hotspot=name file=0]`name` of the
-event, the [hotspot=data file=0]`data` it contains, and the
+Broadcasting events is handled in the [hotspot=broadcastData file=0]`broadcastData()` method.
+First, it checks whether the [hotspot=broadcaster file=0]`broadcaster` value is [hotspot=notNull file=0]`null`.
+There must be at least one subscriber or there's no client to send the event to.
```

Comment:
suggested edit for clarity:

There must be at least one subscriber or there's no client to send the event to.
--->
The `broadcaster` value must include at least one subscriber or there's no client to send the event to.